yt/frontends/ramses/data_structures.py

```diff
@@ -583,6 +605,7 @@ def read_rhs(cast):
             for n in range(rheader['ncpu']):
                 dom, mi, ma = f.readline().split()
                 self.hilbert_indices[int(dom)] = (float(mi), float(ma))
+        f.close()
```

Comment:
It might be simpler to maintain if you make handling the file opening and closing use a context manager. I'd move the in-line definition of `read_rhs` out of the with block though.